test/TestPrometheusMiddleware.AspNetCore.2.1/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="Program.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace TestPrometheusMiddleware.AspNetCore._2._0
```

Comment:
did you create an actual unit tests? You can follow the lead of https://github.com/census-instrumentation/opencensus-csharp/tree/develop/test/OpenCensus.Collector.AspNetCore.Tests